tweb/src/main/frontend/src/scss/fundrequest/_form.scss

```diff
@@ -162,3 +162,22 @@
     }
 }
 
+.placeholder-gray::-webkit-input-placeholder {
```

Comment:
No need for a new cssClass at this point. It can be added to .form-control (or to be a bit more specific .form-control[placeholder]